src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -89,11 +89,10 @@ const UserSelect: FC<UserSelectProps> = ({ name }) => {
                   user={userByAddress || field.value}
                   userName={userDisplayName}
                   size="xs"
-                  className={
-                    usersOptions.isRecipientNotVerified
-                      ? 'text-warning-400'
-                      : ''
-                  }
+                  className={clsx({
+                    'text-warning-400': usersOptions.isRecipientNotVerified,
+                    'text-gray-900': !usersOptions.isRecipientNotVerified,
```

Comment:
What happens if no-one in the Colony is verified and they are not verified?

This is hard to test as all members are showing as verified.

![image](https://github.com/JoinColony/colonyCDapp/assets/33682027/d834e09c-b3c0-49b4-a750-7435f9aafc0c)

Reply:
I am not sure if I understand. If someone selects an unverified user the text will be yellow otherwise it will be `text-gray-900`.

Reply:
The verified icon is showing as of my screenshot, however, I don't have any verified members in the Colony. And based on your logic, you are showing the warning color if `usersOptions.isRecipientNotVerified` is `false`.

Reply:
I can try to fix that, but this logic is not created by me, I have just updated color class names